src/flash/com/longtailvideo/jwplayer/view/View.as

```diff
@@ -119,14 +119,7 @@ public class View extends Sprite {
 
     private function resizeMedia(width:Number, height:Number):void {
         if (_mediaLayer.numChildren > 0 && _model.media.display) {
-            var preserveAspect:Boolean = (_model.fullscreen && _model.stretching === Stretcher.EXACTFIT);
```

Comment:
That's the exact thing that we are trying to fix in this ticket. We want to make it actually do exactfit when we set it to exactfit, instead of changing to uniform.